core/types/rlp_backwards_compat.libevm_test.go

```diff
@@ -138,34 +142,48 @@ func TestBodyRLPBackwardsCompatibility(t *testing.T) {
 	for _, tx := range txMatrix {
 		for _, u := range uncleMatrix {
 			for _, w := range withdrawMatrix {
-				bodies = append(bodies, &Body{tx, u, w})
+				body := &Body{
+					Transactions: tx,
+					Uncles:       u,
+					Withdrawals:  w,
+				}
+				bodies = append(bodies, body)
 			}
 		}
 	}
 
 	for _, body := range bodies {
 		t.Run("", func(t *testing.T) {
-			t.Logf("\n%s", pretty.Sprint(body))
-
```

Comment:
Moved that to error messages; there is no point flooding the logs with this if no error occurs

Reply:
Given the size of the formatted `Body`, I think debugging messages are clearer if you leave them as they were and conditionally log this:

```go
t.Cleanup(func(){
  if t.Failed() {
    t.Logf("\n%s", pretty.Sprint(body))
  }
})
```

Otherwise, for example on a `NoError`, you end up with a lot of distracting noise when the first thing you want is the error.